sec1_intro_and_setup/test_app/config/environments/production.rb

```diff
@@ -0,0 +1,120 @@
+require "active_support/core_ext/integer/time"
```

Comment:
Whats this file for?

Reply:
The `test` environment is used when running automated tests.

Reply:
Your answer doesn't explain the usage of this file.

Reply:
This folder lets you configure how your application will work in production environments.